logic/block.js

```diff
@@ -414,9 +416,9 @@ Block.prototype.getBytes = function(block) {
 		}
 
 		byteBuffer.writeInt(block.numberOfTransactions);
-		byteBuffer.writeLong(block.totalAmount);
-		byteBuffer.writeLong(block.totalFee);
-		byteBuffer.writeLong(block.reward);
+		byteBuffer.writeLong(block.totalAmount.toString());
```

Comment:
Previously numbers, now strings - it doesn't change the behavior of `byteBuffer.writeLong`?

Reply:
writeLong internally calls `writeInt64` and handles the Long value accordingly, here is the gist
```
if (typeof value === 'number')
                    value = Long.fromNumber(value);
                else if (typeof value === 'string')
                    value = Long.fromString(value);
```